about.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose?styles=44178" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>About</title>
+</head>
+
+<body>
+    <header>
+        <div id="heading">
+            <div class="headingImg">
+                <img src="img/logos/facebook.png" class='fb' alt="Facebook logo image">
+                <img src="img/logos/twitter.png" class="twt" alt="twitter logo image">
+            </div>
+            <ul class="headingList">
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <nav class="mobileMenu">
+            <img src="img/buttons/mobile_menu.png" id="abtnImg" alt="Hamburger type button to display menu">
+            <a href="index.html">
+                <div class="logo hide">
+                    <img src="img/logos/icon.png" alt="3d Logo">
+                    <p class="logoText">Developer <br> Summit 2022</p>
+                </div>
+            </a>
+            <ul class="navList hide">
+                <li><a href="about.html" class="listAnchor">About</a></li>
```

Comment:
- [ ] Good job changing the menu link to home as the reviewer suggested however, kindly consider doing the same on the desktop version so the About link on the navbar will be Home link

Reply:
![image](https://user-images.githubusercontent.com/105387310/190860796-3338526a-34c7-409c-972d-7eed64de0e75.png)
Now it looks like this
![image](https://user-images.githubusercontent.com/105387310/190860864-b1cf3d92-7bd8-4eff-ac0c-82dd59b7630d.png)